protocol/app/ante/timestampnonce.go

```diff
@@ -0,0 +1,55 @@
+package ante
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	ante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	accountpluskeeper "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/keeper"
+)
+
+// TODO: combine increment sequence and sequence verification into one decorator
+// https://github.com/cosmos/cosmos-sdk/pull/18817
+type IncrementSequenceDecorator struct {
+	ak ante.AccountKeeper
+}
+
+func NewIncrementSequenceDecorator(ak ante.AccountKeeper) IncrementSequenceDecorator {
+	return IncrementSequenceDecorator{
+		ak: ak,
+	}
+}
+
```

Comment:
Add a comment saying this is forked from x/auth/ante/sigverify.go

Reply:
Will add in upcoming PR